src/status.js

```diff
@@ -0,0 +1,41 @@
+export default class Status {
+  constructor() {
+    if (localStorage.getItem('taskList') === null) {
+      this.taskList = {
+        0: {
+          description: 'task1',
+          completed: false,
+          index: 0,
+        },
+        1: {
+          description: 'task2',
+          completed: false,
+          index: 1,
+        },
+        2: {
+          description: 'task3',
+          completed: false,
+          index: 2,
+        },
+      };
+    } else {
+      this.taskList = JSON.parse(localStorage.getItem('taskList'));
+    }
+  }
+
+    saveStorage = () => {
+      localStorage.setItem('taskList', JSON.stringify(this.taskList));
+    }
+
+    mark = (index) => {
```

Comment:
- [ ] Please add a event listener (`change`) to the checkbox to track the changes. The task object's values for `completed` is updated upon user actions.